### PR TITLE
new test cases to pkg/serviceaccount/util.go

### PR DESCRIPTION
**Pull request by tianshapjq**
**What this PR does / why we need it**:
new test cases to pkg/serviceaccount/util.go

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/serviceaccount/util_test.go [hunk 1]
```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2014 The Kubernetes Authors.
```
Comment (ericchiang) [inline pkg/serviceaccount/util_test.go]: 2017

File: pkg/serviceaccount/util_test.go [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+				Type: v1.SecretTypeServiceAccountToken,
+				ObjectMeta: metav1.ObjectMeta{
+					Annotations: map[string]string{
+						v1.ServiceAccountNameKey: "Secret-name-key",
+					},
+				},
+			},
+			inputSa: &v1.ServiceAccount{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: "ServiceAccount-name-key",
```
Comment (ericchiang) [inline pkg/serviceaccount/util_test.go]: nit: names are always lowercased. no upper case https://kubernetes.io/docs/concepts/overview/working-with-objects/names/

**Comment by tianshapjq** [comment]
@ericchiang thanks for your review

**Comment by tianshapjq** [comment]
close since another pr had been merged

**Pull request closed without merging**